src/main/java/org/jenkinsci/plugins/awsbeanstalkpublisher/extensions/AWSEBS3Setup.java

```diff
@@ -68,10 +69,16 @@ public String getExcludes() {
         return excludes;
     }
 
-    private final Boolean overwriteExistingFile;
+    private final boolean overwriteExistingFile;
```

Comment:
I am not sure this is safe because fields don't get initialized from storage.  This will cause issues for projects that upgrade without setting the value.